src/main/java/uk/gov/hmcts/darts/common/entity/base/CreatedModifiedBaseEntity.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.darts.common.entity.base;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.MappedSuperclass;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import uk.gov.hmcts.darts.common.entity.UserAccountEntity;
+
+import java.time.OffsetDateTime;
+
+@MappedSuperclass
+@Getter
+@Setter
+public class CreatedModifiedBaseEntity {
+
+    @CreationTimestamp
+    @Column(name = "created_ts")
+    private OffsetDateTime createdDateTime;
+
+    @ManyToOne
+    @JoinColumn(name = "created_by", referencedColumnName = "usr_id")
```

Comment:
I don't think we need to have the `referencedColumnName = "usr_id"` for the join